src/pages/lib-v1/auth/emailpassword/q/platform/[platform].mdx

```diff
@@ -69,49 +92,61 @@ User attributes are not required by default. You can make user attributes requir
 
-Once you have these decisions in mind you are ready to enable sign-up, sign-in, and sign-out.
+Once you have these decisions in mind, you are ready to enable sign-up, sign-in, and sign-out.
 
 ## Add the sign-up, sign-in, and sign-out capabilities
 
 You can add the functionality to enable sign-up, sign-in, and sign-out with the Amplify Libraries. You can connect this to an existing UI or create your own.
 
-For many apps, user sign-up and sign-in with a username/password is all that is required. Once authenticated, the app can talk to an API to access and mutate data. 
+For many apps, user sign-up and sign-in with a username and password is all that is required. Once authenticated, the app can talk to an API to access and mutate data. 
+
+<InlineFilter filters={["js"]}>
 
-We will use the username/password authentication flow to get started. Both [third party social provider federation](/lib-v1/auth/social/q/platform/js/) and [Multi-factor authentication](/lib-v1/auth/mfa/q/platform/js/) can be added after this initial set up.
+We will use the username and password authentication flow to get started. Both [third-party social provider federation](/lib-v1/auth/social/q/platform/js/) and [Multi-factor authentication](/lib-v1/auth/mfa/q/platform/js/) can be added after this initial setup.
+
+</InlineFilter>
+
+<InlineFilter filters={["react-native"]}>
+
+We will use the username and password authentication flow to get started. Both [third-party social provider federation](/lib-v1/auth/social/q/platform/react-native/) and [Multi-factor authentication](/lib-v1/auth/mfa/q/platform/react-native/) can be added after this initial set up.
+
+</InlineFilter>
 
 ### Sign-up
 
-You can create a new user in your backend by passing a username, password, and other attributes to `signUp()`.
+You can create a new user in your backend by passing a username, password, and other attributes to `Auth.signUp()`.
 
 <BlockSwitcher>
 <Block name="TypeScript">
 
 ```ts
-import { signUp } from 'aws-amplify/auth';
+import { Auth } from 'aws-amplify';
 
 type SignUpParameters = {
   username: string;
   password: string;
   email: string;
-  phone_number: string;
+  phoneNumber: string;
 };
 
-async function handleSignUp({ username, password, email, phone_number }: SignUpParameters) {
+export async function signUp({ username, password, email, phoneNumber }: SignUpParameters) {
```

Comment:
`phoneNumber` should not be camelcase because of the OIDC attribute name [format](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes)